### DISCUSSION TITLE
Selecting  terminal links exclusively

### DISCUSSION THREAD

**Discussion by fgrunewald** [marrink-lab/vermouth-martinize#490] [Q&A]
Is there a way to apply a link only to the terminals of a polymer via the .ff file for the following scenario:
```
case
1: A-B-B-B-B-A  -> link is not applied  
2: B-B-B-B-B-B  -> link is applied but only to the termini
```
Using the current ff format I can define non-edges like so:
```
[ link ]
resname "B"
[ atoms ] 
B { }
+B { }
[  non-edges ]
B +B
```
The link is correctly applied to the termini in case 2, but is also applied to the second to last B in case 1. Is there anything I'm missing in the syntax that could provide this. @pckroon

**Comment by pckroon** [comment]
```
[ link ]
resname "B"
[ atoms ] 
B { }
+B { }
[  non-edges ]
B +B
B +A
```
maybe? There's a reason why we started using Modifications for termini, rather than the non-edges links though...
